app/src/main/java/de/danoeh/antennapod/adapter/AllEpisodesRecycleAdapter.java

```diff
@@ -211,16 +217,6 @@ public int getItemCount() {
         return itemAccess.getCount();
     }
 
-    public FeedItem getItem(int position) {
-        return itemAccess.getItem(position);
-    }
-
-    public int getPosition() {
-        int pos = position;
-        position = -1; // reset
```

Comment:
@mfietz added this `reset` line in 2015 (419e449b6fac956ca15c3785c77c003c62c524ef). Looks like there was a bug that applied an action on the wrong item. I am not sure if it would still be a better idea to reset the item. Contrarily, a getter method with side effects looks odd to me.

Reply:
I am unclear on the original intent of the reset. The fix, by virtue of removing `getPosition()` (and track `selectedItem` instead), no longer has such issue.